### PR TITLE
adding show default leaderboard

### PR DESCRIPTION
**Pull request by vimask**
Let's add contributing files, thanks
add the function to displaying a leaderboard
https://developers.google.com/games/services/android/leaderboards

### CODE REVIEW

File: lib/play_games.dart
Comment (luanpotter) [inline lib/play_games.dart]: just add a space here for proper formatting:
`'showLeaderboard', {'leaderboardId': leaderboardId}`
after the comma

**Review by luanpotter — COMMENTED** [review]
Hi, @vimask , thanks a lot for your contribution! Just one comment, regarding formatting, but everything else is great. Once you fix that we can merge right away.

**Comment by luanpotter** [comment]
About the contributing file, normally we use the changelog, just adding a line in the new version explaining the feature and who helped with

**Comment by vimask** [comment]
> About the contributing file, normally we use the changelog, just adding a line in the new version explaining the feature and who helped with

Hi @luanpotter, 
`Let's add contributing files` - I mean let's merge my committed files, it is not a requirement for recognition.
Sorry for my English

**Review by luanpotter — APPROVED** [review]
LGTM

**Comment by luanpotter** [comment]
Don't worry, I've added to the changelog with your name :)

**Comment by luanpotter** [comment]
Released 0.5.1, thanks a lot for your contribution!

**Comment by krizda** [comment]
Does not work if LeaderBoard is opened twice in same Widget

**Pull request merged**
